sparse/file.go

```diff
@@ -33,195 +31,56 @@ func SetupFileIO(direct bool) {
 		fileOpen = fio.OpenFile
```

Comment:
It's weird to have private function pointers. It should be done using struct interface. E.g. https://github.com/rancher/longhorn/blob/master/types/types.go#L34